cypress/integration/rancid_tomatillos_spec.js

```diff
@@ -45,29 +45,29 @@ describe('Rancid Tomatillos', () => {
   });
 
   it('Should be able to click on a movie to see its details', () => {
-    cy.fixture('mockMovieData').then(( data ) => {
-      cy.intercept('https://rancid-tomatillos.herokuapp.com/api/v2/movies/632618', {
+    cy.get('.movie[href="/movies/632618"]')
+    .click()
+    .fixture('mockMovieData').then(( data ) => {
```

Comment:
So mysterious that this only works when the `fixture` comes after the first `get` and `click`.